### ISSUE TITLE
Lua Error on Login

### ISSUE THREAD

**Issue by LiangYuxuan** [mynian/ConvertRatings#37]
ConvertRatings version: v2.2.0

Enountered with lua error on login
```
2x Interface/AddOns/ConvertRatings/ConvertRatings.lua:1741: 'end' expected (to close 'if' at line 1738) near 'elseif'


Locals:
```

**Comment by TinyTrbl** [comment]
I have 3 of them:

`2x FrameXML\Bindings.xml:1 Interface/AddOns/ConvertRatings/ConvertRatings.lua:1 Interface/AddOns/ConvertRatings/ConvertRatings.lua:1741: 'end' expected (to close 'if' at line 1738) near 'elseif'
`


`2x Interface/AddOns/ConvertRatings/ConvertRatings.lua:1 Interface/AddOns/ConvertRatings/ConvertRatings.lua:1741: 'end' expected (to close 'if' at line 1738) near 'elseif'
`



`2x Interface/AddOns/ConvertRatings/ConvertRatings.lua:1741: 'end' expected (to close 'if' at line 1738) near 'elseif'
Locals:`

**Comment by leifan8440** [comment]
Line 1740
else -> end

**Comment by Angryaboutelves** [comment]
Thanks for that. Changing that line gives me these. 

`3x FrameXML\Bindings.xml:1 Interface/AddOns/ConvertRatings/ConvertRatings.lua:1 Interface/AddOns/ConvertRatings/ConvertRatings.lua:2165: '<eof>' expected near 'end`

`3x Interface/AddOns/ConvertRatings/ConvertRatings.lua:1 Interface/AddOns/ConvertRatings/ConvertRatings.lua:2165: '<eof>' expected near 'end'`

`3x Interface/AddOns/ConvertRatings/ConvertRatings.lua:2165: '<eof>' expected near 'end'`

**Comment by mynian** [comment]
yea that's not the whole issue, apparently somehow some lines got deleted nowhere near where I made changes .....

**Comment by mynian** [comment]
should be fixed in v2.2.1
